src/binary/uint.rs

```diff
@@ -73,9 +69,56 @@ impl UInt {
     }
 }
 
+/// The big-endian, compact slice of bytes for a `u64`. Leading zero octets are
+/// not part of the representation.
```

Comment:
I think this should be described in terms of the `UInt` encoding primitive (with a link to [that section of the spec](https://amzn.github.io/ion-docs/docs/binary.html#uint-and-int-fields)) rather than in terms of `u64`.

Reply:
Hehe, this is @almann 's suggested change from earlier!

Latest commit should be a good compromise.